app/decorators/controllers/solidus_marketplace/spree/admin/reports_controller_decorator.rb

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
-require 'csv'
+require "csv"
```

Comment:
we try to use double quotes only whenever we need to interpolate variables, leaving the rest as single quotes

Reply:
That is my editor playing with me